lib/govuk_web_banners/recruitment_banner.rb

```diff
@@ -0,0 +1,28 @@
+class RecruitmentBanner
+  def self.for_path(path)
+    all_banners.find do |banner|
+      return banner if banner.page_paths.include?(path)
+    end
+  end
+
+  def self.all_banners
+    recruitment_banners_urls_file_path = Rails.root.join(__dir__,
```

Comment:
No - it's because we're in a gem, and we need to be clear that the reference is relative to the gem. (I don't know if there's some magic alternative to `__dir__`, but I couldn't find one)